Constellation/Node/Main.hs

```diff
@@ -86,7 +87,10 @@ run cfg@Config{..} = do
             }
     ast <- mustLoadPublicKeys cfgAlwaysSendTo
     logf' "Initializing storage {}" [cfgStorage]
-    storage <- berkeleyDbStorage cfgStorage
+    storage <- case break (== ':') cfgStorage of
+        ("bdb", ':':s) -> berkeleyDbStorage s
+        ("dir", ':':s) -> directoryStorage s
+        _              -> berkeleyDbStorage cfgStorage  -- Default
     -- storage <- memoryStorage
```

Comment:
Is there any reason not to include this as an option for users with the new command line options?